include/flatbuffers/flatbuffers.h

```diff
@@ -2251,6 +2297,8 @@ class Verifier FLATBUFFERS_FINAL_CLASS {
     return Verify(veco, byte_size);
   }
 
+  // clang-format off
+  #ifndef FLATBUFFERS_NO_STL
```

Comment:
It seems this code block doesn't use any `std` types.
The `FLATBUFFERS_NO_STL` is used as `NO_VECTOR_OF_STRINGS`.
Is it possible to reduce number of `FLATBUFFERS_NO_STL` by excluding only unnecessary code?

Reply:
Took another pass and reduced it as much as possible.